website/docs/guides/working-with-guides/cloud/1-setting-up-examples-on-cloud.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Setting up example projects in dbt Cloud"
+id: 1-setting-up-examples-on-cloud
+slug: working-with-guides/cloud/1-setting-up
+description: Learn how to set up Guides example projects in dbt Cloud.
+displayText: dbt Cloud setup
+hoverSnippet: Learn how to set up Guides example projects in dbt Cloud.
+---
+
+## Getting started
+
+1. 🔀 **Fork the repo**. Go to the repo specificed as the example project in the guide you're working through, or start from [the base template](https://github.com/dbt-labs/jaffle-shop-template). Click `Use this template` and `Create new repository`.
+2. 🆕 **Create a new project** in dbt Cloud.
+   1. 🏭 **Select your warehouse**. If you don't have a warehouse, we highly suggest either working through the [GitHub Codespaces path](/guides/working-with-guides/core/1-setting-up-examples-on-core-recommended) **OR** first working through the [BigQuery Quickstart](docs/get-started/getting-started/getting-set-up/setting-up-bigquery), which can help you set up a free account on Google's BigQuery warehouse platform. If you choose the latter, use the same credentials you connect to from that Quickstart to connect your new project.
```

Comment:
e,  the Github code or te BigQuery quickstart to set up a free account on Google's BigQuery platform. If you opt for the latter, use the same credentials to connect your new project as you did for the Quickstart.

```suggestion
   1. 🏭 **Select your warehouse**. If you don't have one, either go through the [GitHub Codespaces path](/guides/working-with-guides/core/1-setting-up-examples-on-core-recommended) **OR** try the [BigQuery Quickstart](docs/get-started/getting-started/getting-set-up/setting-up-bigquery), which can help you set up a free account on Google's BigQuery warehouse platform. If you opt for the latter, use the same credentials to connect your new project as you did for the Quickstart.

```